src/codegen/compile.ml

```diff
@@ -7241,6 +7241,9 @@ module FuncDec = struct
         G.nop
       end
       begin
+        if !Flags.trap_on_call_error then
+          E.trap_with env (Printf.sprintf "could not perform %s" purpose)
```

Comment:
```suggestion
          E.trap_with env (Printf.sprintf "could not perform %s" purpose)
```
Hmm, maybe it would be nice if we could preserve the message for throw errors too....

Reply:
Done